### ISSUE TITLE
clang-format fails on MSYS

### ISSUE THREAD

**Issue by akihikodaki** [msys2/MSYS2-packages#5097]
### Description / Steps to reproduce the issue

1. `pacman -S clang git`
2. Open "MSYS2 MSYS"
3. `git clone https://github.com/virtio-win/kvm-guest-drivers-windows`
4. `cd kvm-guest-drivers-windows`
5. `clang-format Balloon/app/device.cpp`

### Expected behavior

Outputs formatted source code. For your information, `mingw-w64-x86_64-clang` works fine.

### Actual behavior

Outputs the following:
```
YAML:5:3: error: unexpected scalar
  Enabled:         true
  ^
YAML:5:3: error: invalid boolean
  Enabled:         true
  ^
Error reading /c/Users/me/source/repos/kvm-guest-drivers-windows/.clang-format: Invalid argument
```

### Verification

- [X] I have verified that my MSYS2 is up-to-date before submitting the report (see https://www.msys2.org/docs/updating/)

### Windows Version

MSYS_NT-10.0-22631

### Are you willing to submit a PR?

No

**Comment by Biswa96** [comment]
Does the required feature depend on clang/llvm version? Because the msys clang is too older than mingw clang https://packages.msys2.org/search?q=clang

**Comment by akihikodaki** [comment]
> Does the required feature depend on clang/llvm version? Because the msys clang is too older than mingw clang https://packages.msys2.org/search?q=clang

Yes. The offending one is `AlignConsecutiveMacros`. It used to be a bool scalar value, but it is now an object that includes `Enabled` and other properties.

**Comment by Biswa96** [comment]
I am not sure if msys clang will be updated soon. It is only for bootstrapping mingw clang toolchain, IIRC.

**Comment by mati865** [comment]
It will be updated, not exactly soon, but it will. Rust, for one, requires it.
